apps/cms/README.md

```diff
@@ -181,3 +181,25 @@ The homepage of the provided UI depicts a featured Pack, which takes prominence
 To update the data model you can start by locally using the Directus UI. But to ensure other developers can apply those changes you will need to write a migration file based on [knex](https://knexjs.org/). See the existing migration files for examples.
 
 Running `npm run bootstrap` will apply new migrations. Additionally, the `package.json` contains other migration scripts (up, down, rollback, and latest) in case they're needed.
+
+### Importing data
+
+To import new data into the CMS, follow the steps below. This will not override existing data in the CMS.
+
+1. Create CSV file for collection(s) with data to import.
+
+Run `npm run import-create-starters` to output documents for each collection inside the `scripts/import-data/starters/` folder. These XLS documents will include all fields for the collection as columns (the field ID is the column header i.e., `preview_image`). There must be a column for every field associated with the collection, but only required fields must have data, otherwise it can be left empty.
+
+Open these documents in the editor of your choosing. Fill out information for each of the items you'd like to import per collection.
+
+For any image fields, you can either specify a UUID for an existing photo in the CMS OR specify a filename (name + extension). The filename would reference a file added to the `scripts/import-data/images` folder. You will be prompted to provide the column header (i.e., the field ID) for each of the image fields, so that it will know to check for any filenames for upload.
+
+If the status is not provided, it will be assigned the 'draft' status.
+
+2. Place new CSV file inside the `scripts/import-data` folder.
```

Comment:
Is the thinking here that these files be added to source control so the data can be synced across devs?

Reply:
...just noticed the gitignore. IMO it'd be better to either commit this file or simply provide it on the command line (https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/)